doc/input-output-reference/src/overview/group-variable-refrigerant-flow-equipment.tex

```diff
@@ -535,6 +535,10 @@ \subsubsection{Outputs}\label{outputs-039}
   HVAC,Average,VRF Heat Pump Heat Recovery Status Change Multiplier {[]}
 \item
   HVAC,Average,VRF Heat Pump Simultaneous Cooling and Heating Efficiency {[}Btu/h/W{]}
+\item
+  HVAC,Average,VRF Heat Pump Heat Recovery Rate {[W]}
+\item
+  HVAC,Sum,VRF Heat Pump Heat Recovery Energy {[J]}
```

Comment:
Added new report variable so users would not have to post-process results and that heat recovery would be included in table reports.